game/scripts/npc/abilities/techies_land_mines.txt

```diff
@@ -50,7 +50,7 @@
       "03"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "damage"                                          "200 380 560 740 1480 2220"
+        "damage"                                          "225 400 575 750 1500 2250" //oaa
```

Comment:
Gives early game techies a small boost, allowing him to farm creeps a bit better and makes enemy punished a tad bit more for not clearing mines. Also numbers pretty :D

Reply:
oaa needs to be uppercase (OAA)